readme.md

```diff
@@ -127,6 +127,26 @@ Custom promise module to use instead of the native one.
 - Pify has useful options like the ability to handle multiple arguments (`multiArgs`).
 - Pify does not have [magic behavior](https://nodejs.org/api/util.html#util_custom_promisified_functions) for certain Node.js methods and instead focuses on predictability.
 
+#### How can I promisify a single class method?
+
+Class methods are not bound, so when they're not called on the class itself, they don't have any context. You can either promisify the whole class or use `.bind()`.
+
+```js
+const pify = require('pify');
+const SomeClass = require('./some-class');
+
+const someClass = new SomeClass();
+
+// `someFunction` can't access its class context.
+const someFunction = pify(someClass.someFunction);
+
+// `someFunction` is extracted from the entire promisified class, allowing it to keep its context.
+const someFunction = pify(someClass).someFunction;
```

Comment:
I don't think this is correct. It will have the same problem as without `pify`.
```suggestion
const someClassPromisified = pify(someClass);
someClassPromisified.someFunction();
```